packages/vue-vuetify/src/complex/ArrayControlRenderer.vue

```diff
@@ -78,14 +78,17 @@
             <tbody>
               <tr
                 v-for="(element, index) in control.data"
-                :key="`${control.path}-${index}`"
+                :key="`${control.path}-${control.data.length}-${index}`"
```

Comment:
Why adapt this key?

Reply:
same as above - to make sure that if the data size changes then the key will be unique enough so that it does not reuse UI elements that is not supposed to